src/main/java/ro/siit/trips/controllers/HomeController.java

```diff
@@ -0,0 +1,37 @@
+package ro.siit.trips.controllers;
+
+
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+@Controller
+public class HomeController {
+    @GetMapping(path = "/login")
+    public String login() {
+        return "login";
+    }
+
+    @RequestMapping(value = {"/logout"}, method = RequestMethod.POST)

```

Comment:
I think spring security would have done logout this way by default (except maybe not POST.. but GET)